### ISSUE TITLE
I can't modify my file content in a row when i update to version8.5.7

### ISSUE THREAD

**Issue by irine1210** [iterate-ch/cyberduck#14419]
/**Describe the bug**
When I modify the code twice in a row, only the first time can be saved successfully, and the content cannot be saved the second time

**To Reproduce**
1. Go to a file such as train.py
2. Click on external editor and use a python IDE to modify my code in  train.py
3. Save and close my train.py code
4. The first time your changing in external editor IDE will be saved. But if you modify it again, you will see that: what i modify in external editor IDE did not saved（which means when I modify the code twice in a row, only the first time can be saved successfully, and the content cannot be saved the second time )

**Expected behavior**
 the content should be saved in the second time too！

**Screenshots**
<img width="296" alt="image" src="https://user-images.githubusercontent.com/55341820/227511018-12661564-aea0-43f7-a09b-960c0ad43036.png">

**Desktop (please complete the following information):**
 - OS: [macOS]
 - Version [8.5.7]

**Log Files**
sorry i did not save log because i delete the version8.5.7 and download the old version8.5.0 and this problems do not exist 

**Additional context**
I redownload the old version(8.5.0) and the problems disappear. so it must be something wrong in  8.5.7
In short, my problem is that : When I modify the code twice in a row, only the first time can be saved successfully, and the content cannot be saved in the other times
Thank you for your time and your excellent work, i love Cyberduck very much🦆
hope my Issues will help you to improve your project  :)
